lightning/src/ln/channelmanager.rs

```diff
@@ -2462,6 +2473,7 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 	/// irrevocably committed to on our end. In such a case, do NOT retry the payment with a
 	/// different route unless you intend to pay twice!
 	///
+	/// Provide recipient_info to include payment_secret or payment_metadata), note that the 
```

Comment:
Looks half-updated?

Reply:
typo on the `(` but I meant for the sentence to flow to the next line. Wrapped to try to match the style from the file. Any tips for better wording?